lib/db-utils.js

```diff
@@ -2,21 +2,36 @@ const dotenv = require("dotenv");
 dotenv.config();
 const { generateKey } = require("../data/dataDefaulter");
 const faunadb = require("faunadb"),
-    q = faunadb.query;
+    fq = faunadb.query;
+const lodash = require("lodash");
 
-var client = new faunadb.Client({ secret: process.env.FAUNA_DB_DEV });
+DEBUG = false; // set to false to disable debugging
+function debug_log() {
+    if (DEBUG) {
+        console.log.apply(this, arguments);
+    }
+}
 
 /* 
     General utility methods, used just within db-utils and test file.
 */
 async function faunaQuery(query) {
+    const client = process.env.DEVELOPMENT
+        ? new faunadb.Client({ secret: process.env.FAUNA_DB_DEV })
```

Comment:
Personally, I prefer having one key FAUNA_DB and just having it set correctly in each environment. What do you think?

Reply:
That's fine with me. It just means that before I merge this, we should change the Github secret from `FAUNA_DB_DEV` and `FAUNA_DB_PROD` to just `FAUNA_DB`. And temporarily any test runs will fail.

Reply:
wait - perhaps I misunderstood... maybe you meant keep the github secrets the same, just have one env variable. this makes sense to me.